samples/constants.go

```diff
@@ -0,0 +1,12 @@
+// Constants
```

Comment:
A special call-out to numeric consts being arbitrary precision and untyped might be useful in this sample: https://gobyexample.com/constants ?

Reply:
Totally missed that. Good call!

Reply:
ok, I don't actually get this. https://goplay.space/#_qqctP3436M
leaving it out for now.
but as @camh-anz noted: type conversion happens on printing probably.